templates/repo/issue/view_content/sidebar.tmpl

```diff
@@ -335,6 +335,91 @@
 					</div>
 				{{end}}
 			</div>
+
+			{{ if .IsRepoAdmin }}
+			<div class="ui divider"></div>
+			<div class="ui watching">
```

Comment:
Can we find another place for this button since to sidebar seems to overloaded to me. Maybe at the comment form?